### PR TITLE
Initial pull request to add intro to RNN tutorial

### PR DESCRIPTION
**Pull request by roverman**
Initial draft of "Intro to RNN tutorial", including the jupyter notebook for the tutorial, and an image file for the diagram of simple RNN layer.

### CODE REVIEW

**Comment by lamberta** [comment]
Preview: http://colab.research.google.com/github/roverman/docs/blob/master/site/en/tutorials/keras/Intro_to_RNN.ipynb

Thanks, Chunxiang!
Is the use of `keras.preprocessing.sequence` necessary (or much more convenient than the alternative)?

**Comment by roverman** [comment]
Hi Bill,
Thanks for the review.  
The keras.preprocessing.sequence import is for the sequence padding function.  I think we need to pad the sequence to the same length for the batch processing. 
Which alternative did you mean?

File: site/en/tutorials/keras/Intro_to_RNN.ipynb [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+    "## A recurrent layer in Keras"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Keras recurrent layers can be run in two different modes: they return either the full sequences of successive outputs for each timestep (a 3D tensor of shape (batch_size, timesteps, output_features)), or return only the last output for each input sequence (a 2D tensor of shape (batch_size, output_features)). These two modes are controlled by the return_sequences constructor argument.\n",
+    "\n",
+    "Let's take a look at an example that uses a SimpleRNN layer and returns only the output at the last timestep:\n",
```
Comment (mari-linhares) [inline site/en/tutorials/keras/Intro_to_RNN.ipynb]: Maybe finish this sentence with '.' instead of ':'?

Comment (mari-linhares) [inline site/en/tutorials/keras/Intro_to_RNN.ipynb]: Maybe finish this sentence with '.' instead of ':'?

File: site/en/tutorials/keras/Intro_to_RNN.ipynb [hunk 2]
```diff
@@ -0,0 +1,441 @@
...
+    "model.add(SimpleRNN(32, return_sequences=True))\n",
+    "model.add(SimpleRNN(32))  # This last layer only returns the last outputs.\n",
+    "model.summary()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now let's try to use such a model on the IMDB movie review classification problem. First, let's preprocess the data:"
```
Comment (mari-linhares) [inline site/en/tutorials/keras/Intro_to_RNN.ipynb]: First, let's load and preprocess the data. The preprocessing consists basically of padding the sentences so that all have the same length. If the RNN is dynamic as in [tf.nn.dynamic_rnn](https://www.tensorflow.org/api_docs/python/tf/nn/dynamic_rnn) this step is not needed.
